shell.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```

should we add `set -e` here?

---

@guewen I do not see the interest. 
We may add a trapp to hide error to customer and log something useful for us